scripts/get_build_deps.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+
+# Add the package sources
+#wget -qO - https://packages.irods.org/irods-signing-key.asc | sudo apt-key add -
+#echo "deb [arch=amd64] https://packages.irods.org/apt/ $(lsb_release -sc) main" | sudo tee /etc/apt/sources.list.d/renci-irods.list
+sudo apt-get update
+sudo apt install -y irods-dev irods-externals-\* irods-icommands
+
+# log in so we can grab the up to date irods lol
+cat bison_credentials | iinit
+
+# Grab the temporary files.
+iget -r  /tempZone/home/rods/ub20packages_for_violet/ubuntu-20.04
```

Comment:
Tell me about this.

Reply:
Same.  Temporary because of switch user.  Current implementation is dependent on 4.3.1-to-be.